pkg/vpa/vpa.go

```diff
@@ -258,3 +264,20 @@ func (r Reconciler) createVPA(namespace string, vpaName string) error {
 	}
 	return nil
 }
+
+func vpaUpdateModeForNamespace(ns *corev1.Namespace) v1beta2.UpdateMode {
+	for k, v := range ns.GetLabels() {
+		if strings.ToLower(k) != vpaUpdateModeLabel {
+			continue
+		}
+		switch v {
```

Comment:
I noticed you have only implemented two of the four updateModes available. Is there a reason for that?

Reply:
It looks like we only implemented exactly what we were looking to have access to. I also don't see us having added these additional update modes in the current master of our fork. I'll submit a commit to add the other modes too.

- [] Support all VPA update modes